Scripts/RemediationScripts/Remediate-SetSQLServerMinReqTLSVersion.ps1

```diff
@@ -0,0 +1,962 @@
+        Write-Host $([Constants]::SingleDashLine)
+    }   
+}
+
+function Reset-SQLServerRequiredTLSVersion
+{
+     <#
+        .SYNOPSIS
+        Rolls back remediation done for 'Azure_SQLServer_DP_Use_Secure_TLS_Version' Control.
+
+        .DESCRIPTION
+        Rolls back remediation done for 'Azure_SQLServer_DP_Use_Secure_TLS_Version' Control.
+        Resets minimal TLS Version on the production slot and all non-production slots in all SQL Servers in the Subscription. 
+        
+        .PARAMETER SubscriptionId
+        Specifies the ID of the Subscription that was previously remediated.
+        
+        .PARAMETER Force
+        Specifies a forceful roll back without any prompts.
+        
+        .Parameter PerformPreReqCheck
+        Specifies validation of prerequisites for the command.
+        
+        .Parameter ExcludeNonProductionSlots
+        Specifies exclusion of non-production slots from roll back.
+        
+        .PARAMETER FilePath
+        Specifies the path to the file to be used as input for the roll back.
+
+        .INPUTS
+        None. You cannot pipe objects to Reset-SQLServerRequiredTLSVersion.
+
+        .OUTPUTS
+        None. Reset-SQLServerRequiredTLSVersion does not return anything that can be piped and used as an input to another command.
+
+        .EXAMPLE
+        PS> Reset-SQLServerRequiredTLSVersion -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck -FilePath C:\AzTS\Subscriptions\00000000-xxxx-0000-xxxx-000000000000\202109131040\setMinTLSVersionForSQLServers\RemediatedSQLServers.csv
+
+        .EXAMPLE
+        PS> Reset-SQLServerRequiredTLSVersion -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck -ExcludeNonProductionSlots -FilePath C:\AzTS\Subscriptions\00000000-xxxx-0000-xxxx-000000000000\202109131040\setMinTLSVersionForSQLServers\RemediatedSQLServers.csv
+
+        .LINK
+        None
+    #>
+
+    param (
+        [String]
+        [Parameter(Mandatory = $true, HelpMessage="Specifies the ID of the Subscription that was previously remediated.")]
+        $SubscriptionId,
+
+        [Switch]
+        [Parameter(HelpMessage="Specifies a forceful roll back without any prompts")]
+        $Force,
+
+        [Switch]
+        [Parameter(HelpMessage="Specifies validation of prerequisites for the command")]
+        $PerformPreReqCheck,
+
+        [String]
+        [Parameter(Mandatory = $true, HelpMessage="Specifies the path to the file to be used as input for the roll back")]
+        $FilePath
+    )
+
+    Write-Host $([Constants]::DoubleDashLine)
+    Write-Host "`n[Step 1 of 4] Preparing to reset SQL Server TLS Version in Subscription: $($SubscriptionId)"
+    Write-Host $([Constants]::SingleDashLine)
+    if ($PerformPreReqCheck)
```

Comment:
change it according to standards and as in above function

Reply:
done